### PR TITLE
Possibly fixes tgs discord bot, might also fix instancing

### PR DESCRIPTION
**Pull request by Carbonhell**

### CODE REVIEW

**Comment by Kmc2000** [comment]
I CAN DREAM

File: code/__DEFINES/tgs.config.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #define TGS_EXTERNAL_CONFIGURATION
-#define TGS_V3_API
+//#define TGS_V3_API
```
Comment (Kmc2000) [inline code/__DEFINES/tgs.config.dm:2]: Seems to work, instancing didn't happen on two seperate reboots and data now works.

**Review by Kmc2000 — APPROVED** [review]

**Comment by Kmc2000** [comment]
based carbon

**Comment by Cyberboss** [comment]
HOW

**Comment by Cyberboss** [comment]
jUST FUCKING HOW

**Comment by Cyberboss** [comment]
I CANT EVEN AND I WROTE IT

**Comment by Cyberboss** [comment]
S E PE R A T E MO DU L E S

**Comment by Kmc2000** [comment]
REEEE

**Comment by francinum** [comment]
This didn't end up fixing anything.

**Comment by francinum** [comment]
Production environment reverted to V3 pending fix.

**Pull request merged**